src/status_im/chat/models/loading.cljs

```diff
@@ -76,9 +76,9 @@
 
 (rf/defn handle-mark-all-read-in-community-successful
   {:events [::mark-all-read-in-community-successful]}
-  [{:keys [db] :as cofx} chat-ids]
+  [{:keys [db] :as cofx} {:keys [chats] :as messenger-response}]
```

Comment:
`messenger-response` is unused.

Reply:
yes, i knew, just want to make it more clear that `chats` comes from where, WDYT? @ilmotta

Reply:
<img src="https://github.com/status-im/status-mobile/assets/46027/e0a930db-8738-4c51-90f4-317ddd92e022" width="700" />

@qfrank, in the screenshot I changed the code to use `reg-event-fx` just as a reminder that code inside `rf/defn` is not linted and that's why you have a green build.

If you want to give meaning to the destructured map, you can use `_messenger-response` or you can simply not destructure. In the mobile team we seem to be obsessed with destructuring as if it's a rule (you get feedback in PRs if you don't use it), but I've seen many Clojure devs intentionally not destructuring so that they can explicitly give a name to the argument.

Pick your poison 😉 

```clojure
;; With underscore
(rf/defn handle-mark-all-read-in-community-successful
  {:events [::mark-all-read-in-community-successful]}
  [{:keys [db] :as cofx} {:keys [chats] :as _messenger-response}]
  (rf/merge cofx
            {:db       (reduce mark-chat-all-read db (map :id chats))
             :dispatch [:activity-center.notifications/fetch-unread-count]}))

;; Without destructuring
(rf/defn handle-mark-all-read-in-community-successful
  {:events [::mark-all-read-in-community-successful]}
  [{:keys [db] :as cofx} messenger-response]
  (rf/merge cofx
            {:db       (reduce mark-chat-all-read db (map :id (:chats messenger-response)))
             :dispatch [:activity-center.notifications/fetch-unread-count]}))
```

Reply:
I prefer the first one!